eng/testing/tests.mobile.targets

```diff
@@ -151,22 +151,20 @@
       AssemblySearchPaths="@(AssemblySearchPaths)" />
   </Target>
 
-  <UsingTask TaskName="DownloadTimeZoneData"
-             AssemblyFile="$(CreateCreateWasmBundlesAssemblyPath)" />
+  <Import Project="$(MonoProjectRoot)wasm\timezones.props" Condition="'$(TargetOS)' == 'Browser'" />
+  <UsingTask TaskName="CompileTimeZoneData"
+             AssemblyFile="$(GenerateWasmBundlesAssemblyPath)" />
```

Comment:
Should we move this to `timezones.props`?

Reply:
Move the Timezone compilation to `timezones.props`? That makes sense.

Reply:
I wouldn't move that over as you usually define UsingTask statements in target files.

Reply:
Ok then, I will keep it as is.